pkgs/development/compilers/elm/default.nix

```diff
@@ -106,6 +106,18 @@ let
             ln -sf ${elm-instrument}/bin/elm-instrument unpacked_bin/elm-instrument
           '';
         };
+
+      create-elm-app = patchBinwrap [elm] (nodePkgs.create-elm-app.override {
+        preRebuild = ''
+          # This should not be needed (thanks to binwrap* being nooped) but for some reason it still needs to be done
+          # in case of just this package
+          sed 's/\"install\".*/\"install\":\"echo no-op\",/g' --in-place node_modules/elmi-to-json/package.json
```

Comment:
Seems like you need `patchBinwrap [ elm elmi-to-json ]` or it doesn't work?

Reply:
elm compiler doesn't use `binwrap` package but own custom script in `install.js`.

Reply:
Sorry I was too sloppy checking the context.

I believe I tried to use patchBinwrap function but it didn't work. I'm afraid something might have changed in `binwrap` which broke our script but let me double check that.

Reply:
@domenkozar i got it resolved. I would swear this wasn't working before but most likely I just did some stupid mistake - probably I was applying patch for `elm` instead of `elmi-to-json` or something.